rebasehelper/tests/conftest.py

```diff
@@ -59,3 +61,18 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mocked_spec_object(monkeypatch, spec_attributes):
+    spec_attributes['save'] = lambda: None
```

Comment:
How can one redefine the `save` method now? This is not setting default value, this is overriding any passed one.